<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>surgeryandroom</class>
 <widget class="QDialog" name="surgeryandroom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>580</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Gargi</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Операции и операционные</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="3">
       <widget class="QPushButton" name="btnNext">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Следующая</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnPrevious">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Предыдущая</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbSpecialization_7">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Gargi</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Отделение</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbSpecialization_8">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Gargi</family>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Номер операционной</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leOperatingRoomNumber">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leDepartment">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnBack">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Назад</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnAdd">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Добавить </string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnDel">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnSave">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="lbSpecialization_9">
     <property name="minimumSize">
      <size>
       <width>650</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>650</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Gargi</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Операционная</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="5" colspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="lbSpecialization_10">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Gargi</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Проводимые операции</string>
       </property>
       <property name="margin">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnDel2">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnSave2">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QTableWidget" name="twSurgery">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>610</width>
         <height>250</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnAdd2">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Добавить </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
